src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -962,7 +957,14 @@ protected boolean generateEntriesPaymentRequest(PaymentRequestDocument preq, Lis
 
           
         }
-
+        
+        if (CollectionUtils.isNotEmpty(preq.getItems())) {
+            LOG.info("generateEntriesPaymentRequest, cancel previous revisions of preq document: " + preq.getDocumentNumber());
+            purapAccountRevisionService.cancelPaymentRequestAccountRevisions(preq.getItems(), preq.getPostingYearFromPendingGLEntries(), 
+                    preq.getPostingPeriodCodeFromPendingGLEntries());
+        } else {
+            LOG.info("generateEntriesPaymentRequest, unable to cancel previous revisions becuase they items are empty for document: " + preq.getDocumentNumber());
```

Comment:
Minor typo in label; should be changed from "...becuase they items..." to "...because the items..." instead.

Reply:
ok, will fix